app/controllers/application_controller.rb

```diff
@@ -7,6 +7,8 @@ class ApplicationController < ActionController::Base
   attr_reader :current_local_authority
   helper_method :current_local_authority
 
+  default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
I note that we have this, but also have explicilty set the builder on the `form_with` in `app/views/drawings/index.html.erb`. Is one of these redundant?